### PR TITLE
Add release for bionic base image

### PR DESCRIPTION
**Pull request by chenbh**

### CODE REVIEW

**Comment by zachgersh** [comment]
@chenbh - are you going to drop the wip off of this?

Also, I assume you are going to go make changes to the concourse-ci pipeline to change the dockerfile location?

**Comment by chenbh** [comment]
@zachgersh yep, the main pipeline change is at concourse/concourse#3364, I'll let you know once that's merged

I guess that instead of marking it as `wip`, it would've been better to leave a comment saying it's blocked until that PR is in

File: dockerfiles/ubuntu/Dockerfile
Comment (cirocosta) [inline dockerfiles/ubuntu/Dockerfile:21]: ~~Hey, Is there any reason why we'd need to run the tests twice given that at the `builder` phase we already run it? thxx!~~

I didn't previously see that it's just building the tests there. Thanks @YoussB for explaining!

**Review by cirocosta — CHANGES_REQUESTED** [review]
Hey, It'd be nice to have the README reflecting the fact that now we have multiple images being built (with updated instructions for that).

It might also be the case that we don't need to have the tests being copied in the `test` stage and then run there - wdyt?

Thanks!

**Review by cirocosta — APPROVED** [review]
SGTM

**Pull request merged**
